src/Pages/welcomePage/Welcome.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import "./Welcome.css";
+import Bulb from '../../images/Bulb.jpg';
+const Welcome = () => {
+    return(
+    <div className="welcomepage">
+      <div className = "contentW">
+        <div className="Para0">Your Ultimate Power Partner</div>
+        <div className="Para1">
+          Powering over 1 million homes, we are one of the largest electricity
```

Comment:
this actually is a para. Though you can write texts directly in a div tag, the recommended practice is to use <p> or <span> where deemed appropriate.

Refer: https://stackoverflow.com/questions/23967257/text-inside-div-tags-semantically-correct-or-not